tests/framework/behaviors/CacheableWidgetBehaviorTest.php

```diff
@@ -38,36 +38,6 @@ protected function setUp()
         $this->initializeWidgetMocks();
     }
 
-    public function testDefaultCache()
-    {
-        $this->assertEquals('cache', $this->simpleWidget->cache);
-    }
-
-    public function testDefaultDuration()
-    {
-        $this->assertEquals(60, $this->simpleWidget->cacheDuration);
-    }
-
-    public function testDefaultDependency()
-    {
-        $this->assertNull($this->simpleWidget->cacheDependency);
-    }
-
-    public function testDefaultKey()
-    {
-        $this->assertNull($this->simpleWidget->cacheKey);
-    }
-
-    public function testDefaultKeyVariations()
-    {
-        $this->assertEquals([], $this->simpleWidget->cacheKeyVariations);
-    }
-
-    public function testDefaultEnabled()
```

Comment:
@developeruz why would you delete a test which ensures that the `cacheEnabled` property is actually initialized with `true` by default?

Reply:
Because on line 64 `public $cacheEnabled = true;` `cacheEnabled` set with `true` and imho here no needs to check if I haven't passed anything, that default value is still there.

Reply:
@developeruz following this logic we can delete the whole test-suite of the framework. Here is the code https://github.com/yiisoft/yii2, it makes the framework work.

This test covers line `64` you are referring to and ensures somebody does not remove the default value in future.

Reply:
Removing test in this case is OK.

Reply:
@Kolyunya, removing default value is violating backward compatibility, that is why no one won't remove the default value in future without being sure (will add a test in that time) that the same default value set in another way.
PS: sorry for intervention in your PR. I thought it is abandoned one and I just want to help this PR to be merged. Anyway, you did a great job 👍